internal/subscriber/retry/delayconsumer_test.go

```diff
@@ -20,6 +22,39 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func setupDC(t *testing.T) (
+	ctx context.Context,
+	ctrl *gomock.Controller,
+	subCtx context.Context,
+	cancel context.CancelFunc,
+	store *mocks.MockIBrokerStore,
+	handler *mocks3.MockMessageHandler,
+	consumer *mocks2.MockConsumer,
+	cache *cachemock.MockICache,
+) {
+	ctx = context.Background()
+	subCtx, cancel = context.WithCancel(ctx)
+	ctrl = gomock.NewController(t)
+	store = mocks.NewMockIBrokerStore(ctrl)
+	handler = mocks3.NewMockMessageHandler(ctrl)
+	consumer = mocks2.NewMockConsumer(ctrl)
+	cache = cachemock.NewMockICache(ctrl)
+	cache.EXPECT().Get(gomock.AssignableToTypeOf(subCtx), gomock.Any()).Return([]byte{'0'}, nil).AnyTimes()
+	cache.EXPECT().Set(gomock.AssignableToTypeOf(subCtx), gomock.Any(), gomock.Any(), gomock.Any()).Return(nil).AnyTimes()
+	handler.EXPECT().Do(gomock.AssignableToTypeOf(subCtx), gomock.Any()).Return(nil).AnyTimes()
+	consumer.EXPECT().CommitByPartitionAndOffset(gomock.Any(), gomock.Any()).Return(messagebroker.CommitOnTopicResponse{}, nil).AnyTimes()
+	consumer.EXPECT().Resume(gomock.Any(), gomock.Any()).Return(nil).AnyTimes()
+	consumer.EXPECT().Resume(gomock.AssignableToTypeOf(subCtx), gomock.Any()).Return(nil).AnyTimes()
```

Comment:
Why is Resume mocked twice within the scope of the same function with `AnyTimes()` ? Shouldn't it be a single EXPECT call?